src/services/lease-service/get-tenant.ts

```diff
@@ -0,0 +1,161 @@
+import { Contact, Lease } from 'onecore-types'
```

Comment:
It looks like this file mainly uses queries towards xpand? Could we move it to the xpand folder to make  that clear?

Reply:
Not sure where to put it. The xpand folder is inside adapters and I don't think it's really an adapter fn. Its more of a service i guess since it combines many different existing functions? 🤷